sktime/utils/parallel.py

```diff
@@ -62,6 +63,8 @@ def parallelize(fun, iter, meta=None, backend=None, backend_params=None):
           If ``n_jobs`` is not passed, it will default to ``-1``, other parameters
           will default to ``joblib`` defaults.
         - "dask": any valid keys for ``dask.compute`` can be passed, e.g., ``scheduler``
+        - "logger_name": allows for setting up a custom logger when using ray as backend
```

Comment:
the strings at the start explain which keys are valid based on the `backend` string, i.e., a bullet point you add here should start with `- "ray"`